doc/source/rllib.rst

```diff
@@ -0,0 +1,112 @@
+RLLib: Ray's modular and scalable reinforcement learning library
```

Comment:
I changed my mind about modular, how about just scalable :)

Reply:
Sounds good to me

Reply:
Yeah, I also don't think we should claim modular just yet ;)

Reply:
Oh, also we need to document how to use it in the cluster setting :)